nixos/tests/postal.nix

```diff
@@ -0,0 +1,113 @@
+{ pkgs, lib, ... }:
+
+{
+  name = "postal";
+
+  meta.maintainers = with lib.maintainers; [ MatthieuBarthel ];
+
+  # tests are broken on aarch64 on ofborg
+  meta.broken = pkgs.stdenv.hostPlatform.isAarch64;
```

Comment:
I personally don't feel comfortable merging this with tests being marked broken on aarch64.

Reply:
You're right I will try to solve this one more time

Reply:
@xanderio tests are now ok, I disabled systemd-notify for postal-web in the tests (the issue you spotted previously), but I never had this issue using the module on ARM hosts so I believe it's not a real issue.

I also removed some tests that were probably a bit fancy and were failing on ofborg (but not on a PI 5 ^^).